packages/firestore/src/local/indexeddb_query_cache.ts

```diff
@@ -114,15 +114,7 @@ export class IndexedDbQueryCache implements QueryCache {
     transaction: PersistenceTransaction,
     queryData: QueryData
   ): PersistencePromise<void> {
-    return this.saveQueryData(transaction, queryData).next(() => {
-      return this.retrieveMetadata(transaction).next(metadata => {
-        if (this.updateMetadataFromQueryData(queryData, metadata)) {
-          return this.saveMetadata(transaction, metadata);
-        } else {
-          return PersistencePromise.resolve();
-        }
-      });
-    });
+    return this.saveQueryData(transaction, queryData);
```

Comment:
You will still potentially need to update the metadata. Once `QueryCache` starts tracking the highest sequence number, it is likely that an update will trigger a metadata write.

Reply:
I updated the PR to allow future Greg to set the sequence number as part of the `setLastRemoteSnapshotVersion` call. Let me know if that will work as is for GC.

I am also still accepting nominations for the API name voting challenge.